### PR TITLE
Added information on how to flash firmware without re-pairing

### PR DESCRIPTION
**Pull request by Bruceforce**
As discussed in #906 I tried to make a small documentation on how to flash the firmware of the coordinator without re-pairing all devices.

If there is something wrong it would be nice if you let me know.

### CODE REVIEW

File: docs/information/flashing_without_re-pairing.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Flashing without re-pairing
+
+## Why do I need to re-pair?
+Flashing firmware usually requires re-pairing of all connected [routers](zigbee_network.md#Router).
+If you remove the coordinator form the network, the remaining routers (e.g. bulbs) can still communicate with each other and they still form a zigbee network.
```
Comment (h4nc) [inline docs/information/flashing_without_re-pairing.md]: from

File: docs/information/flashing_without_re-pairing.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+If you now flash the coordinator and try to bring it back to the network, it will use its old `pan_id` incremented by one. 
+So if you used the default `pand_id: 0x1a62` your coordinator will now use `pan_id: 0x1a63`.
```
Comment (h4nc) [inline docs/information/flashing_without_re-pairing.md]: pand_id --> pan_id

Comment (Bruceforce) [reply docs/information/flashing_without_re-pairing.md]: Thanks 👍

**Comment by Koenkk** [comment]
could you add this to `index.md`? Create a pr for this.

**Comment by Bruceforce** [comment]
> could you add this to `index.md`? Create a pr for this.

Sorry, I'm confused. Do you want me to create a new PR just for the added link to `index.md`?

**Comment by Koenkk** [comment]
NVM, Please apply this change: https://github.com/Koenkk/zigbee2mqtt/commit/f6c1fff9a833b9b4d12ef8bf51dfc2f290cfd62b

**Comment by Koenkk** [comment]
Thanks!

**Comment by dlashua** [comment]
This doesn't make it clear at which point the coordinator needs to be outside of the network.

Just during flashing? I.E., I can drive to the park with my laptop, flash the device, and then bring it home again, plug it back in where it usually lives and start up zigbee2mqtt and it'll work? Or, do I need to plug it in and have zigbee2mqtt running while it's outside of the network?

This seems like something that should be able to be handled by software either way. Not being able to upgrade without either driving to a remote location or disabling every routing device in my home is a pretty big drawback, especially considering the constantly-changing nature of this project and how often that means reflashing might be required.

**Comment by Bruceforce** [comment]
I thought 
> Once you are outside the network range you can [flash the coordinator with the new firmware](../getting_started/flashing_the_cc2531.md) and start zigbee2mqtt. *After zigbee2mqtt started successfully, you can shutdown zigbee2mqtt and bring the coordinator back in network range*.

Would be clear enough. It says you should start with Zigbee2mqtt once. So driving to park just to flash the coordinator would not be enough. But if this is unclear for you it would be great if you file a PR to make it more obvious.

In the second point you are right if course, but I think it would be hard to do via software, since (as explained in the PR) your devices still form a valid network after your coordinator is gone. So the coordinator would need to join the old network, but since there is no coordinator to make the join happen this can't be done I think.

**Pull request merged**